### PR TITLE
🎨 New front-end statics

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- 🎨  New statics exposed
    - `SESSION_COOKIE_MAX_AGE`
    - `SIMCORE_VCS_RELEASE_TAG`  (provided by the CD)
    - `SIMCORE_VCS_RELEASE_URL`
- 🎨 search users by email can be done with a glob  pattern e.g. `GET v0/users:search?email=crespo*`

This is an example of statics
```json
 {
 "apiVersion": "0.40.2",
 "appName": "simcore_service_webserver",
 "buildDate": "2022-01-09T12:26:29Z",
 "vcsRef": "dd536f998",
 "vcsUrl": "git@github.com:ITISFoundation/osparc-simcore.git",
 "vcsReleaseTag": "staging_ResistanceIsFutile12",
 "vcsReleaseUrl": "https://github.com/ITISFoundation/osparc-simcore/releases/tag/staging_ResistanceIsFutile12",
 "stackName": "master-simcore",
 "webserverLogin": {
  "LOGIN_ACCOUNT_DELETION_RETENTION_DAYS": 31
 },
 "webserverSession": {
  "SESSION_COOKIE_MAX_AGE": 604800
 },
 "webserverProjects": {
  "PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES": 5
 },
 "pluginsDisabled": [
  "WEBSERVER_CLUSTERS",
  "WEBSERVER_SCICRUNCH",
  "WEBSERVER_VERSION_CONTROL",
  "WEBSERVER_EXPORTER"
 ]
}
```
NOTE that if the values are set to None, the fields are note exported. For instance, if SESSION_COOKIE_MAX_AGE=None (i.e. there is no expiration) then 
```json
{
 "webserverSession": { },
}
```

## Related issue/s

- feedback from @odeimaiz 

## How to test

- `test_application_settings`

## Dev-ops checklist ⚠️

- [x] New environ `SIMCORE_VCS_RELEASE_TAG`
- [x] Corresponding [osparc-config MR](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/495)

### CODE REVIEW

**Review by jsaq007 — APPROVED** [review]
👍

File: services/web/server/src/simcore_service_webserver/users/_api.py [hunk 1]
```diff
@@ -62,12 +62,19 @@ async def set_user_as_deleted(app: web.Application, user_id: UserID) -> None:
     )
 
 
+def _glob_to_sql_like(glob_pattern):
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/users/_api.py:65]: could you please add some types here?

**Review by GitHK — APPROVED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]

**Comment by YuryHrytsuk** [comment]
Could you point me to the place where `SIMCORE_VCS_RELEASE_TAG` is defined?

**Review by mrnicegyu11 — APPROVED** [review]
lgtm thanks

**Comment by odeimaiz** [comment]
Merci!

**Pull request merged**